identity/app/views/fragments/form/switchboardLabel.scala.html

```diff
@@ -21,9 +22,12 @@
     <div class="account__switchboardLabelContent">
         @fragments.form.checkbox(field, Checkbox(field).args:_*)
         <div class="manage-account__switchboardCheckbox"></div>
-        <p class="manage-account__switchboardTitle">
+        <h3 class="manage-account__switchboardTitle">
             @title
-        </p>
+            @if(subheading.isDefined){
+                <strong>@subheading</strong>
+            }
```

Comment:
You are displaying the `Option` variable rather than its content here. Is this on purpose?
If not you would need to get its content: `@subheading.map(s => <strong>s</strong>).getOrElse("")`

Reply:
right now it's seemingly working like this but i'll update the syntax!

Reply:
https://github.com/playframework/twirl/pull/160